server/pom.xml

```diff
@@ -5,12 +5,11 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>1.0.0</version>
-		<relativePath/> <!-- lookup parent from repository -->
+		<version>3.3.0</version>
```

Comment:
Think you changed the version in the wrong place. This parameter is for specifying the version of spring-boot to use.